bindings/python/src/OpenSpaceToolkitPhysicsPy/Time/Instant.cpp

```diff
@@ -55,7 +56,8 @@ inline void                     OpenSpaceToolkitPhysicsPy_Time_Instant      (
         .def("get_modified_julian_date", &Instant::getModifiedJulianDate)
         .def("get_leap_second_count", &Instant::getLeapSecondCount)
         .def("to_string", +[] (const Instant& anInstant) -> String { return anInstant.toString() ; })
-        .def("to_string", +[] (const Instant& anInstant, const Scale& aScale) -> String { return anInstant.toString(aScale) ; })
+        .def("to_string", +[] (const Instant& anInstant, const Scale& aScale) -> String { return anInstant.toString(aScale) ; }, arg("scale"))
+        .def("to_string", +[] (const Instant& anInstant, const Scale& aScale, const DateTime::Format& aDateTimeFormat) -> String { return anInstant.toString(aScale) ; }, arg("scale"), arg("date_time_format"))
```

Comment:
Two comments on this:

1. There's a better way now to overload bindings with Pybind11, see [docs](https://pybind11.readthedocs.io/en/stable/classes.html#overloaded-methods). Search for the OSTk codebase for usage, it's actually simpler than using lambdas (this approach dates back from when we were using Boost Python instead).
2. The second overload doesn't actually use `aDateTimeFormat`, is this expected?

Reply:
Excellent. Agreed that's a better syntax, and yes I totally brain farted on not using the variable.